### DISCUSSION TITLE
v0.3.5.1

### DISCUSSION THREAD

**Discussion by dr-bonez** [Start9Labs/start-os#2534] [Announcements]
## Highlights
* Revert perpetual performance mode for quieter fan
* Minor bug fixes

## What's Changed
* [chore]: fix automated tests by @dr-bonez in https://github.com/Start9Labs/start-os/pull/2506
* rename frontend to web and update contributing guide by @MattDHill in https://github.com/Start9Labs/start-os/pull/2509
* Update README.md by @MattDHill in https://github.com/Start9Labs/start-os/pull/2511
* set better governor hierarchy and add cli command to change by @dr-bonez in https://github.com/Start9Labs/start-os/pull/2519
* FE version bump and minor copy updates by @MattDHill in https://github.com/Start9Labs/start-os/pull/2517
* bump version by @dr-bonez in https://github.com/Start9Labs/start-os/pull/2520
* play song unconditionally by @dr-bonez in https://github.com/Start9Labs/start-os/pull/2518
* allow non-zero exit `cpupower frequency-info -g` by @dr-bonez in https://github.com/Start9Labs/start-os/pull/2523
* retry migration if incomplete by @dr-bonez in https://github.com/Start9Labs/start-os/pull/2522
* redesign firmware updater by @dr-bonez in https://github.com/Start9Labs/start-os/pull/2521
* improve robustness of install by @dr-bonez in https://github.com/Start9Labs/start-os/pull/2524
* add cli to rotate key by @dr-bonez in https://github.com/Start9Labs/start-os/pull/2525
* prevent hanging on reboot by @dr-bonez in https://github.com/Start9Labs/start-os/pull/2526
* set content disposition for cert by @dr-bonez in https://github.com/Start9Labs/start-os/pull/2527
* fix ntp sync status check daemon by @dr-bonez in https://github.com/Start9Labs/start-os/pull/2528
* v0.3.5.1 by @dr-bonez in https://github.com/Start9Labs/start-os/pull/2513

**Full Changelog**: https://github.com/Start9Labs/start-os/compare/v0.3.5...v0.3.5.1

## Checksums
BLAKE-3
```
5e3d6e4865fa3417bbe2a89d9a53181140abcfbc6b7f7eb1ad1be72067e8a73f  startos-0.3.5.1-39de098_aarch64.iso
c8357d0e49962120960e843aa8a77b24bb6a98e2a2a7415902598736fd32d020  startos-0.3.5.1-39de098_aarch64-nonfree.iso
49d635fbcb3d8239a5550158acc21f6d9cd67f1bfc4fae124eac625673ec5a80  startos-0.3.5.1-39de098_raspberrypi.img.gz
afd33094c90faade9f7fa8df1752558d4c708287f3237e638975aceec302bc84  startos-0.3.5.1-39de098_x86_64.iso
92f1a6f88e4da72d081babf9a47bd09d1870ddfb624756f3cd8d1cf2440d9cbd  startos-0.3.5.1-39de098_x86_64-nonfree.iso
```
SHA-256
```
aa3c70b411e5ad2c52ccf927e7bae0394d816d8e197511cbebf0503e23a1a964  startos-0.3.5.1-39de098_aarch64.iso
8ebae051ccff4a99b23326ac40e8c2ce85d40438f2093bb1be967e3aa2961178  startos-0.3.5.1-39de098_aarch64-nonfree.iso
daba5d01f93a53433db2347a81cf46accba9096711eccef9c90836ccc78a7e69  startos-0.3.5.1-39de098_raspberrypi.img.gz
181fec43a3b7287fe6c6cdb70bcba79a5596fc2be1dbe5a54e09a4c142272155  startos-0.3.5.1-39de098_x86_64.iso
29b0f1e0211568ea66d6b729536ff84aed3b6cbc1c38540a7c80a391fd01616e  startos-0.3.5.1-39de098_x86_64-nonfree.iso
```

<hr /><em>This discussion was created from the release <a href='https://github.com/Start9Labs/start-os/releases/tag/v0.3.5.1'>v0.3.5.1</a>.</em>

**Comment by vjfqwq-crypto** [comment]
The SHA-256 hash for the startos-0.3.5.1-39de098_raspberrypi.img.gz does not match. Getting "ea6a1e4770d447cb62d423179e539df1e3b5d6b2468b540469d722a96a4a16af" instead.

**Reply by rjcndev** [reply]
@vjfqwq-crypto 
im getting the correct one

`/Downloads$ sha256sum startos-0.3.5.1-39de098_raspberrypi.img.gz 
daba5d01f93a53433db2347a81cf46accba9096711eccef9c90836ccc78a7e69  startos-0.3.5.1-39de098_raspberrypi.img.gz`

**Reply by waltzWaltz** [reply]
Mine doesn't match either

```
sha256sum startos-0.3.5.1-39de098_x86_64-nonfree.iso 
e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855  startos-0.3.5.1-39de098_x86_64-nonfree.iso
```

**Reply by waltzWaltz** [reply]
I just deleted the file and downloaded again, and got a match.. never had that happen before.

sha256sum startos-0.3.5.1-39de098_x86_64-nonfree.iso 
29b0f1e0211568ea66d6b729536ff84aed3b6cbc1c38540a7c80a391fd01616e  startos-0.3.5.1-39de098_x86_64-nonfree.iso
